### ISSUE TITLE
[HOLD #20471] Mp3 audio file displays differently than video file when returning from offline mode

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#18886]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Go to staging dot on web chrome
2. Go to any chat
3. Send one video mp4 file and the second mp3 audio file
4. Go offline
5. Now click on both the files and notice that they open up in a new tab
6. Now go online
7. Wait for a few seconds
8. After the page is refreshed, notice that the video file plays without problem but the audio file shows some weird display

## Expected Result:
Mp3 file should also be played or should get downloaded in a similar how video file behaves when coming back to online from the offline mode
## Actual Result:
Mp3 file shows some weird display on coming back to online from the offline mode and doesn't behave the same way how video file does
## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.13.1
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/d6c85b54-fbbb-498b-a866-dedef568ced6

https://github.com/Expensify/App/assets/43996225/f51485fb-5758-40be-a9a6-96b1bf59ab51

**Expensify/Expensify Issue URL:**
**Issue reported by:** @priya-zha
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1683904534756129

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01b1830648bef2fa99</li>
        <li>Upwork Job ID: 1658211495048851456</li>
        <li>Last Price Increase: 2023-05-15</li>
    </ul>
</details>

**Comment by therealsujitk** [comment]
This is an **internal** issue, the [`Content-Type`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) header isn't being set before sending the file. For some cases the browser is able to guess what kind of file it is loading, for other cases it needs to know how to display that data.

**Comment by maddylewis** [comment]
reviewed / repo'd / added internal label for review.

**Comment by maddylewis** [comment]
checking in about no proposals yet here - https://expensify.slack.com/archives/C049HHMV9SM/p1684441841209559?thread_ts=1683904534.756129&cid=C049HHMV9SM

**Comment by maddylewis** [comment]
hiya @puneetlath - looking for some direction on this one so i added the eng label. I've added the `internal` label and bumped the original thread to try and get some proposal action, but i haven't had any luck yet. but since it's `internal` maybe i should've added the `engineering` label immediately??

**Comment by puneetlath** [comment]
@maddylewis yeah I think in that case, it makes sense to just go ahead and add the label to get an engineer assigned. I'll start looking into this further soon.

**Comment by maddylewis** [comment]
@puneetlath - what would you like to do this one? let me know if you want me to reach out to Call Stack or SWM - ty!

**Comment by puneetlath** [comment]
Unfortunately, this is most likely a back-end issue, so will need to be handled internally. I just need to find time for it.

**Comment by puneetlath** [comment]
Still haven't had a chance to look into it. Have been focused on SNH.

**Comment by maddylewis** [comment]
puneet is OOO until later this month. moving to weekly until he returns.

**Comment by puneetlath** [comment]
I will try to take this to the finish line next week after SNH.

**Comment by puneetlath** [comment]
I think this would actually get solved by the media player that we are adding in https://github.com/Expensify/App/issues/20471. Going to go ahead and put it on hold for that.

**Comment by maddylewis** [comment]
still holding for this one - https://github.com/Expensify/App/issues/20471

**Comment by maddylewis** [comment]
still holding for - https://github.com/Expensify/App/issues/20471

**Comment by maddylewis** [comment]
i believe we are still holding for - https://github.com/Expensify/App/issues/20471

**Comment by puneetlath** [comment]
Still on hold. @maddylewis I'm actually going to un-assign myself for now. Feel free to re-assign the engineering label once this is ready for some action.

**Comment by maddylewis** [comment]
holding/moving to monthly - https://github.com/Expensify/App/issues/20471#issuecomment-1672380987

**Comment by maddylewis** [comment]
still holding
